pkg/printer/alphabetical.go

```diff
@@ -0,0 +1,58 @@
+// alphabetical.go are the functions to get a sorted list of keys from sysls map types
+// Unfortunately go doesn't allow generics and pointer types don't implement empty interfaces
+// so map[string]interface{} doesn't cut it
+
+package printer
+
+import (
+	"sort"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+func alphabeticalAttributes(m map[string]*sysl.Attribute) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+func alphabeticalApplications(m map[string]*sysl.Application) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	return keys
+}
+
+func alphabeticalEndpoints(m map[string]*sysl.Endpoint) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		if k != "" {
```

Comment:
Any reason why the empty string check only needs to be done for endpoint names?

Reply:
Encountered some strange behaviour where sometimes there were empty endpoint names

Reply:
Think we can do the check for all of the above^ Helps make it consistent at least, so that all empty appnames param names etc are ignored?